bin/commands/.help

```diff
@@ -1,3 +1,9 @@
-A command line utility helps you managing Zowe instance.
+Synopsis
+    zwe [sub-command [sub-command]...] [parameter [parameter]...]
```

Comment:
I believe `Synopsis` section can be automatically generated.

This variable `${ZWE_CLI_COMMANDS_LIST}` is the sub-command list. Then we can always append `[parameter [parameter]...]` after it.

Reply:
Also I feel `Name` and `Synopsis` seems duplicated info to me. What do you think?

Reply:
You are right that `Name` and `Synopsis` seems duplicate for deep commands.

But In most of the man or help pages command title is always displayed first before any other explanation so Instead of keeping it under `Name` heading it can be displayed individually.

Let me know if it looks redundant to you I can remove it.

Reply:
It looks good to me. Thanks Aditya!